client-app/src/components/DraftOrcaDashboard.js

```diff
@@ -76,8 +76,8 @@ const DraftOrcaDashboard = () => {
     axios
       .post("http://localhost:5001/upload", formData)
       .then((response) => {
-        const uploadedFileName = response.data.filename.split("/").pop();
-        setFileName(uploadedFileName);
+        const uploadedFileName = response.data.filename.split(/[/\\]/).pop();
+        setFileName(response.data.filename);
```

Comment:
Please use 'response.data.file_path' to get the file path and assign it to the variable setFileName. If possible, update the variable names setFileName to setFilePath and fileName to filePath throughout the file.